content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -403,6 +410,27 @@ In the past, the annotation `volume.beta.kubernetes.io/mount-options` was used i
 of the `mountOptions` attribute. This annotation is still working, however
 it will become fully deprecated in a future Kubernetes release.
 
+### Node Affinity
+
+A PV can specify [node affinity](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volumenodeaffinity-v1-core) to define constraints that limit what nodes this volume can be accessed from. Only the nodes that satifsy the constraints can access the volume.
```

Comment:
satisfy

Reply:
I would add a section that users normally do not need to set this field.  It is automatically populated for aws, gce, and azure block volume types, and the other major use case is local volumes.  Most other volume types do not need this field.

Reply:
Fixed. Added a note section